### ISSUE TITLE
Quick start not working, crashing without output

### ISSUE THREAD

**Issue by tst-fractal** [expo/expo-three#257]
running npx create-react-native-app -t with-three creates the project
however the npm install does not finish with success unless using --legacy-peer-deps or bumping three version to ^0.127.0" 
After successful npm install when running on the device the example app crashes without any output

**Comment by JSneak** [comment]
Are you using an android device? I also have a similar issue.

**Comment by tst-fractal** [comment]
Yes, I am using an android device (Android 11) , did not test on IOS yet.

**Comment by FladioArmandika** [comment]
I have the same problem with iOS here

**Comment by shadow-boy** [comment]
i think this expo lib is incompatible with app ，android or ios

**Comment by shadow-boy** [comment]
use a physical device as the main web page said

**Comment by caesar4321** [comment]
Downgrade it to expo v43.0.0 and import React from 'react' from the example.

**Comment by princeSmall** [comment]
I also have a similar issue with iOS
